### DISCUSSION TITLE
Housekeeping

### DISCUSSION THREAD

**Discussion by matt-psaltis** [ChilliCream/hotchocolate#4952] [Ideas]
A list of general tidy up or house keeping items found during people's travels in the code base.

- [x] SchemaCoordinateNode - Should Name come before ofDirective?

**Comment by michaelstaib** [comment]
I finally looked into the schema coordinate. The spec puts the OfDirective prop first since that is the parsing direction.

If you have the following `@foo` we have the OfDirective token first `@` then comes the name.
